superset-frontend/src/views/datasetList/DatasetList.tsx

```diff
@@ -193,36 +195,51 @@ class DatasetList extends React.PureComponent<Props, State> {
       }: any) => kind[0]?.toUpperCase() + kind.slice(1),
       Header: t('Type'),
       accessor: 'kind',
+      size: 'lg',
     },
     {
       Header: t('Source'),
       accessor: 'database_name',
+      size: 'lg',
     },
     {
       Header: t('Schema'),
       accessor: 'schema',
+      size: 'lg',
     },
     {
       Cell: ({
         row: {
           original: { changed_on: changedOn },
         },
-      }: any) => <span className="no-wrap">{moment(changedOn).fromNow()}</span>,
+      }: any) => {
+        const momentTime = moment(changedOn);
+        const time = momentTime.format('h:m a');
+        const date = momentTime.format('MMM D, YYYY');
```

Comment:
it would be nice to move these format strings into constants (either in this file or in a more global constant file) assuming that these date and time formats will be widely used in the future